index.js

```diff
@@ -115,6 +115,7 @@ UTP.prototype.bind = function (port, ip, onlistening) {
 }
 
 UTP.prototype.listen = function (port, ip, onlistening) {
+  if (typeof ip === 'function') return this.bind(port, null, ip)
```

Comment:
I needed to add this line in order for the onlistening to be fired if no ip is passed